docs/interaction/rpc.md

```diff
@@ -10,118 +10,508 @@ The following methods are available:
+  - (2) `<sender account id>` (ignored)
+
+```bash
+http post https://rpc.nearprotocol.com jsonrpc=2.0 method=tx params:='["CqiBnYCRgkGV2odJPXJTBkoZHXjDNQFJKw7oRYWU1j6g", ""]' id=dontcare
+```
+
+```json
+{
+  "id": "dontcare",
+  "jsonrpc": "2.0",
+  "result": {
+    "receipts_outcome": [
+      {
+        "block_hash": "52yXcXjL6gR2HV5c5bXdv3FE7uGgcXdCaipq9MQo9kUx",
+        "id": "Cj9eqPrvh9oCAUi6hRxiQjJzsEAa7bQnja3q5U6bX5yC",
+        "outcome": {
+          "gas_burnt": 924119500000,
+          "logs": [],
+          "receipt_ids": [],
+          "status": {
+            "SuccessValue": ""
+          }
         },
-        "transactions": []
+        "proof": [
+          {
+            "direction": "Left",
+            "hash": "8SxoArHHF3ko3RvigCxCHGLtbWg9XiQawQEGraMAZ3yy"
+          }
+        ]
+      }
+    ],
+    "status": {
+      "SuccessValue": ""
+    },
+    "transaction": {
+      "actions": [
+        {
+          "AddKey": {
+            "access_key": {
+              "nonce": 0,
+              "permission": "FullAccess"
+            },
+            "public_key": "ed25519:9y5RuRL7YaGKxsbZynHQjVktVVMCk4fMNywkTYiqdnPz"
+          }
+        }
+      ],
+      "hash": "CqiBnYCRgkGV2odJPXJTBkoZHXjDNQFJKw7oRYWU1j6g",
+      "nonce": 1,
+      "public_key": "ed25519:8xNjM142ibwb79JvY7u6grXFEtEgeaohez1HmoAwC7xT",
+      "receiver_id": "david_sanford80wirethingsnet",
+      "signature": "ed25519:4e3aAi93Fdk4GEnzFTddhnFwHzN8rrSXbkS9sJfbpnn5MJSLFVp8TF2TjQcWXQmtY5Mb9EPJ1KdV91h1UJ2zg5Ld",
+      "signer_id": "david_sanford80wirethingsnet"
+    },
+    "transaction_outcome": {
+      "block_hash": "52yXcXjL6gR2HV5c5bXdv3FE7uGgcXdCaipq9MQo9kUx",
+      "id": "CqiBnYCRgkGV2odJPXJTBkoZHXjDNQFJKw7oRYWU1j6g",
+      "outcome": {
+        "gas_burnt": 924119500000,
+        "logs": [],
+        "receipt_ids": [
+          "Cj9eqPrvh9oCAUi6hRxiQjJzsEAa7bQnja3q5U6bX5yC"
+        ],
+        "status": {
+          "SuccessReceiptId": "Cj9eqPrvh9oCAUi6hRxiQjJzsEAa7bQnja3q5U6bX5yC"
+        }
+      },
+      "proof": [
+        {
+          "direction": "Right",
+          "hash": "81vrGGwyr15GvPkhMeV3NcHZLRy6T2z86JQyeyZPe87r"
+        }
+      ]
     }
+  }
 }
 ```
 
-## Transaction Status
+## Validators Endpoint
+
+`validators(blockHash: string)`
 
-`tx(hash: bytes)`: queries status of the transaction by hash, returns FinalTransactionResult that includes status, logs and result: `{"status": "Completed", "logs": [{"hash": "<hash>", "lines": [], "receipts": [], "result": null}]}`
+Queries for active validators on the network and returns details about them and the current state of validation on the blockchain
 
-`http post http://127.0.0.1:3030/ jsonrpc=2.0 method=tx params:=["<base 58 of transaction hash>", "<sender account id>"] id=dontcare`
+- HTTP `POST`
+- URL `https://rpc.nearprotocol.com`
+- method `validators`
+- params
+  - (1) `<base 58 block hash>` (see [status page](https://rpc.nearprotocol.com/status) for a valid `latest_block_hash`)
```

Comment:
also can use "null" for latest

Reply:
@ilblackdragon 

❓not sure how to get this to work

thrashing with a few formats didn't work: 
- `params:='["null"]'` -- "provided string contained invalid character 'l' at byte 2"
- `params:='[null]'` -- "Failed parsing args: invalid type: null, expected a string"
- `params:='null'` -- "Require at least one parameter"

note: this works just fine
```
http post https://rpc.nearprotocol.com jsonrpc=2.0 method=validators params:='["8ehA3NYL5uSF8zefbnqnz66twYJ45rfst6SrqBNv7oka"]' id=dontcare
```

Reply:
spoke w @frol and found this works fine on staging.  

`params:='[null]'` is correct

Reply:
✅